### PR TITLE
CASMNET-1805 | network upgrades for CANU specific things main

### PR DESCRIPTION
**Pull request by kosonenj**
# Description

Updating CANU related commannds and release notes locations

# Checklist Before Merging

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/network/management_network/generate_switch_configs.md [hunk 1]
```diff
@@ -6,16 +6,18 @@ For example, if there is a suspected configuration issue on single switch, a con
 
 ## Prerequisites
 
-* CANU installed with version 1.1.11 or greater.
+* CANU installed with version 1.6.13 or greater.
```
Comment (GunashekarKM) [inline operations/network/management_network/generate_switch_configs.md:9]:
```suggestion
* CANU installed with 1.6.13 or later versions.
```

File: operations/network/management_network/generate_switch_configs.md [hunk 2]
```diff
@@ -6,16 +6,18 @@ For example, if there is a suspected configuration issue on single switch, a con
...
   * Run `canu --version` to see version.
   * If doing a CSM install or upgrade, a CANU RPM is located in the release tarball. For more information, refer to the [Update CANU From CSM Tarball](canu/update_canu_from_csm_tarball.md) procedure.
+  * Alternatively: Upgrade or install latest version of CANU from GitHub [Install/Upgrade CANU](canu_install_update.md)
```
Comment (GunashekarKM) [inline operations/network/management_network/generate_switch_configs.md:12]:
```suggestion
  * Alternatively, upgrade or install latest version of CANU from GitHub [Install/Upgrade CANU](canu_install_update.md).
```

**Review by lukebates123 — APPROVED** [review]

**Review by mharding-hpe — CHANGES_REQUESTED** [review]
I want to do some linting of this

File: operations/network/management_network/README.md [hunk 3]
```diff
@@ -155,7 +155,8 @@ The following are known open issues with this branch of the software.
 
 The Symptom statement describes what a user might experience if this is seen on the network. The Scenario statement provides additional environment details and trigger summaries. When available, the Workaround statement provides a workaround to the issue.
 
-> No currently known issues.
+> Known and fixed issue can be found from CANU release notes.
```
Comment (mharding-hpe) [inline operations/network/management_network/README.md:158]:
```suggestion
> Known and fixed issue are listed in the CANU release notes.
```

File: operations/network/management_network/README.md [hunk 4]
```diff
@@ -155,7 +155,8 @@ The following are known open issues with this branch of the software.
...
+[CANU releases](https://github.com/Cray-HPE/canu/releases)
```
Comment (mharding-hpe) [inline operations/network/management_network/README.md:159]:
```suggestion
See [CANU releases](https://github.com/Cray-HPE/canu/releases).
```

File: operations/network/management_network/README.md [hunk 5]
```diff
@@ -71,51 +65,42 @@ summary of the network design used in the CSM management network.
 
 See [CSM Automatic Network Utility (CANU)](canu/README.md)
```
Comment (mharding-hpe) [inline operations/network/management_network/README.md:66]:
```suggestion
See [CSM Automatic Network Utility (CANU)](canu/README.md).
```

File: operations/network/management_network/generate_switch_configs.md [hunk 6]
```diff
@@ -1,50 +1,56 @@
...
 The following are the different architectures that can be specified:
 
 * `Tds` – Aruba-based Test and Development System. These are small systems characterized by Kubernetes NCNs cabled directly to the spine.
 * `Full` – Aruba-based Leaf-Spine systems. These are usually customer production systems.
-* `V1` – Dell and Mellanox based systems of either a TDS or Full layout.
+* `V1` – Any Dell and Mellanox-based systems.
 
 Generating a configuration file can be done for a single switch, or for the full system. Below are example commands for both scenarios:
 
 **Important:** Modify the following items in your command:
```
Comment (mharding-hpe) [inline operations/network/management_network/generate_switch_configs.md:37]:
```suggestion
**Important:** Modify the following items in the command:
```

**Review by mharding-hpe — APPROVED** [review]

**Pull request merged**